internal/cmd/config/list/list.go

```diff
@@ -19,8 +20,16 @@ func NewCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list",
 		Short: "Lists the current CLI configuration values",
-		Long:  "Lists the current CLI configuration values.",
-		Args:  args.NoArgs,
+		Long: fmt.Sprintf("%s\n%s\n%s\n%s\n%s\n%s\n%s",
+			"Lists the current CLI configuration values, based on the following sources (in order of precedence):",
+			"- Environment variable",
+			`  The environment variable is the name of the setting, with underscores ("_") instead of dashes ("-") and the "STACKIT" prefix.`,
+			"  Example: you can set the project ID by setting the environment variable STACKIT_PROJECT_ID.",
+			"- Configuration set in CLI",
```

Comment:
Would it be clearer if rephrased to

```suggestion
			"- Configuration set in the config file",
```

Reply:
I phrased this here to abstract from the config file, so you don't have to know of its existence
Basically tried to write "whatever was set using `stackit config set/unset`" (which ends up in the config file)
Not sure if it was the best choice, what do you think?